canonical-web-and-design/vanilla-framework
# Add dark theme to the buttons

## Done

WIP

Demo: https://vanilla-framework-3807.demos.haus/docs/examples/patterns/buttons/dark

Fixes #3779

## QA

- Open [demo](https://vanilla-framework-3807.demos.haus/docs/examples/patterns/buttons/dark)
- Check if dark buttons render as expected
- Review updated documentation:
  - https://vanilla-framework-3807.demos.haus/docs/patterns/buttons#theming
  - https://vanilla-framework-3807.demos.haus/docs/component-status

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)


## Screenshots

<img width="845" alt="Screenshot 2021-06-30 at 17 16 12" src="https://user-images.githubusercontent.com/83575/123986804-ec87b200-d9c6-11eb-9d80-80627f8e493d.png">
